gen/calc.py

```diff
@@ -421,7 +425,10 @@ def validate_os_type(os_type):
         },
         'rexray_config_method': {
             'file': {
-                'must': {'rexray_config_contents': calculate_rexray_config_contents},
+                'must': {'rexray_config_contents': calculate_rexray_config_from_file},
+            },
+            'string': {
+                'must': {'rexray_config_contents': calculate_rexray_config_from_string},
             },
```

Comment:
+1, my only hesitation is that enterprise docs mention it: https://docs.mesosphere.com/1.7/administration/installing/custom/configuration-parameters/

Do we care? I can follow up with a docs PR to remove it if not.